azblob/url_blob.go

```diff
@@ -143,6 +143,16 @@ func (b BlobURL) Delete(ctx context.Context, deleteOptions DeleteSnapshotsOption
 		nil, BlobDeleteNone)
 }
 
+// PermanentDelete permanently deletes-soft deleted snapshots & soft-deleted version blobs and is a dangerous operation and SHOULD NOT BE USED.
```

Comment:
typo: the dash is placed in the wrong place

Reply:
Should not be used unless you know exactly the implications. We will not provide support for this API.